api/tests/unit/environments/identities/test_unit_identities_feature_states_views.py

```diff
@@ -106,3 +112,121 @@ def test_user_with_view_environment_permission_can_retrieve_all_feature_states_f
 
     # Then
     assert response.status_code == status.HTTP_200_OK
+
+
+def test_identity_clone_flag_states_from(
+    project: Project,
+    environment: Environment,
+    admin_client: Client,
+    features_for_identity_clone_flag_states_from: typing.Callable,
+) -> None:
+    # GIVEN
+
+    # Create 3 features
```

Comment:
Again, I know you like to comment things as you go, but for really obvious stuff it's not worth the mental overhead.

Reply:
I don't think 3 words can cause a mental overhead :)

Reply:
It's not the mental overhead so much as the risk of it becoming stale. IMO we should be stringent with comments and use them only when really necessary because it's additional code that needs to be kept up to date when the actual test code changes.

Reply:
SHould I remove all the inline coments?